lib/couchdb.js

```diff
@@ -0,0 +1,99 @@
+var existsSync = require('fs').existsSync
+var join = require('path').join
+
+var _ = require('lodash')
+var async = require('async')
+var jsonfile = require('jsonfile')
+var log = require('npmlog')
+var randomstring = require('randomstring')
+var request = require('request')
+
+module.exports = function (config, callback) {
+  if (!config.db.url) {
+    var storePath = join(config.paths.data, 'config.json')
+    var storeExists = existsSync(storePath)
+    var store = storeExists && jsonfile.readFileSync(storePath, {
+      throws: false
+    }) || {}
+    var secret = store.couch_httpd_auth_secret
+
+    if (!secret) {
+      secret = randomstring.generate({
```

Comment:
In the old hoodie-server we had it that at this point, we already had set up an admin user, and the secret created.

What is the plan for setting up an admin user in the new hoodie-server?